app/templates/components/ad-slot.hbs

```diff
@@ -1,6 +1,6 @@
 <div
   id={{name}}
-  class="ad-slot {{adEngine3ClassName}} {{hiddenClassName}}"
+  class="ad-slot {{adEngine3ClassName}} {{defaultClasses}}"
```

Comment:
Will it work with an array?

Reply:
Nope. This is why there is join(' '): https://github.com/Wikia/mobile-wiki/pull/2409/files#diff-c158578aa157904c3f0096a4f978d60017581fbd5338bb9ebf9b06fd40b0d1d6R195